python/ray/_private/tls_utils.py

```diff
@@ -68,6 +66,7 @@ def generate_self_signed_tls_certs():
 
 
 def add_port_to_grpc_server(server, address):
+    import grpc
```

Comment:
Could use a comment explaining the funky import placing.

Reply:
Thanks, I rolled this one back, it was from an earlier experiment (to fully get rid of grpc) and we are not ready for that (yet) :)